package.json

```diff
@@ -10,7 +10,8 @@
   "scripts": {
     "start": "nps",
     "test": "nps test",
-    "precommit": "lint-staged && opt --in pre-commit --exec \"npm start validate\""
+    "precommit": "lint-staged && opt --in pre-commit --exec \"npm start validate\"",
+    "flow": "flow"
```

Comment:
Could you move this to `package-scripts.js` and add `flow` to the `validate` script?